pkg/client/run_test.go

```diff
@@ -68,3 +72,42 @@ func TestRunInvalidConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestPrintPodStatus(t *testing.T) {
+	expected := []string{
+		"kube-system/coredns-74ff55c5b-cpfqs: Pending: PodScheduled: Unschedulable, 0/1 nodes are available: 1 node(s) had taint {node.kubernetes.io/not-ready: }, that the pod didn't tolerate.",
+		"kube-system/coredns-74ff55c5b-vk77h: Pending: PodScheduled: Unschedulable, 0/1 nodes are available: 1 node(s) had taint {node.kubernetes.io/not-ready: }, that the pod didn't tolerate.",
+		"kube-system/etcd-kind-control-plane: Running",
+		"kube-system/kindnet-hknk7: Running",
+		"kube-system/kindnet-qhzxn: Running",
+		"kube-system/kindnet-x9fsq: Running",
+		"kube-system/kube-apiserver-kind-control-plane: Running",
+		"kube-system/kube-controller-manager-kind-control-plane: Running",
+		"kube-system/kube-proxy-8jj5r: Running",
+		"kube-system/kube-proxy-g79v6: Running",
+		"kube-system/kube-proxy-lrtdp: Running",
+		"kube-system/kube-scheduler-kind-control-plane: Running",
```

Comment:
As my other comment says, most of this test case doesn't add any value. We dont need to check the running state 10 times.